test/data/v17-long-history.json

```diff
@@ -57,15 +57,15 @@
         "goerli": "ok"
       }
     },
-    "BlacklistController": {
+    "BlocklistController": {
```

Comment:
.... what controller is this 🤔 Was this supposed to be `PhishingController`?

Reply:
We could just delete it. It seems safe to assume the test doesn't depend upon it.

Reply:
The BlacklistController was removed from state in migration 21...I'm not sure we should change the instance here to BlockList. 
It is not referencing the phishing controller, this file is only used in one tx-controller test and does not utilize any of the blocking/allowing data. We should simplify that test in another PR

Reply:
Meaning I think we should remove this entire file soon as well.

Reply:
Deleted that part of the mock state in c9dd343